addons/binding/org.openhab.binding.icloud/src/main/java/org/openhab/binding/icloud/internal/json/icloud/Features.java

```diff
@@ -0,0 +1,236 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.icloud.internal.json.icloud;
+
+/**
+ *
+ * @author Patrik Gfeller
+ *
+ */
+public class Features {
+    private boolean MSG;
```

Comment:
What do these things mean, how are they stored in the json? Please map the names.